src/component/dataZoom/SliderZoomModel.ts

```diff
@@ -79,6 +79,8 @@ export interface SliderDataZoomOption extends DataZoomOption, BoxLayoutOptionMix
 
     handleIcon?: string
 
+    handleLabelShow?: boolean
```

Comment:
Just be curious about the position of `Show`.

Reply:
+1. Both `show` and `handle` and be a verb or an noun. `handleLabelShow` could be understand as handle the show of a label. Is `showHandleLabel` more natural?